tensorboard/plugins/interactive_inference/witwidget/pip_package/build_pip_package.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/sh
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+if [ -z "${RUNFILES}" ]; then
+  RUNFILES="$(CDPATH= cd -- "$0.runfiles" && pwd)"
+fi
+
+if [ "$(uname)" = "Darwin" ]; then
+  sedi="sed -i ''"
+else
+  sedi="sed -i"
+fi
+
+set -x
+command -v virtualenv >/dev/null
```

Comment:
this line fails silently when virtualenv is not installed (does not warn that it is not installed, just says build completed successfully).